src/components/App/App.js

```diff
@@ -9,70 +9,84 @@ import SMSNotification from "../SMSNotification/SMSNotification";
 import Search from "../Search/Search";
 import Weather from "../Weather/Weather";
 import Forecast from "../Forecast/Forecast";
-import { WEATHER_API_KEY, WEATHER_API_URL, DISASTER_API_URL } from '../../api'
+import { WEATHER_API_KEY, WEATHER_API_URL, DISASTER_API_URL, ADD_USER_URL} from '../../api'
 import './App.css';
 
 function App() {
 
   const [location, setLocation] = useState("")
+  const [smsFeedback, setSMSFeedback] = useState("")
   const [currentWeather, setCurrentWeather] = useState("")
   const [forecast, setForecast] = useState("")
   const [disasterAlerts, setDisasterAlerts] = useState("")
 
+  const formUserData = (phoneNumber) => {
+    return {
+      "lat": String(location.lat),
+      "long": String(location.lng),
+      "phone": String(phoneNumber),
+    }
+  }
+
+  const addUser = (phoneNumber) => {
+    const userData = formUserData(phoneNumber)
+    fetch(ADD_USER_URL, { 
```

Comment:
There's an opportunity to move all API interactions to an utils file and clean up App.js